examples/ovs-cni.yml

```diff
@@ -1,18 +1,17 @@
----
 apiVersion: extensions/v1beta1
```

Comment:
is it an example or part of installation?

Reply:
It can be used for installation if you want to use kube-system and :latest image.